views/timeline.ejs

```diff
@@ -156,8 +156,14 @@
 
     </div> <!--END: main-grid-->
   </div> <!--END: main-page-container -->
-
-  <script src="/js/characterCount.js"></script>
   <script src="/js/likeChirp.js"></script>
```

Comment:
Did you mean to delete this line since there's a likeChirp.js in the conditional below?

Reply:
Good catch.  I updated timeline.ejs file on this PR such that the likeChirp.js is only inside the conditional.